ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -2208,12 +2207,44 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
-      FILTERS_ADDITIONAL: ~CPaasrunOnly&;Smokerun&
       TEST_SCENARIOS: Windows_Containers
       TEST_TIMEOUT: "50"
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-winc
+- as: azure-ipi-ovn-winc-f14
+  cron: 20 17 16,30 * *
```

Comment:
pls update the cron setting following https://github.com/rrasouli/release/blob/a70458f5302c54a2c5386d9ad8736bb17a28d3e0/ci-operator/config/openshift/openshift-tests-private/README.md#job-frequency